README.md

```diff
@@ -85,6 +89,9 @@ git clone -b trunk --single-branch --depth 1 git@github.com:WordPress/wordpress-
 
 ## Contributing
 
+>WordPress Playground is an ambitious project in its early days. If the feature you need is missing, you are more than welcome to start a discussion, open an issue, and even propose a Pull Request to implement it.
```

Comment:
The failed check is the linter, it seems to require a space after the `>` character:

```suggestion
> WordPress Playground is an ambitious project in its early days. If the feature you need is missing, you are more than welcome to start a discussion, open an issue, and even propose a Pull Request to implement it.
```

However, was that space intended? It seems like the paragraph of text below didn't have one.

Reply:
This para was removed in a future commit, see 493321da639e5cefb589a34d4ab9f73906baac29.